source/lib-intro.tex

```diff
@@ -2338,8 +2337,7 @@
 and
 \indextext{\idxcode{extern ""C++""}}%
 \tcode{extern "C++"}
-linkage,
-\footnote{
+linkage,\footnote{
```

Comment:
Nice catch, thanks! Patch amended, with 1 new visual difference, namely the removal of the space at the start of this footnote.